lib/marked.js

```diff
@@ -460,7 +460,8 @@ var inline = {
   code: /^(`+)\s*([\s\S]*?[^`])\s*\1(?!`)/,
   br: /^ {2,}\n(?!\s*$)/,
   del: noop,
-  text: /^[\s\S]+?(?=[\\<!\[_*`]| {2,}\n|$)/
+  text: /^[\s\S]+?(?=[\\<!\[_*`]| {2,}\n|$)/,
+  moretext: /^(?:[^\s([{]_[A-Za-z0-9]|[A-Za-z0-9]_\S)/
```

Comment:
The point of `moretext` is to support “no intra-emphasis”, the GFM feature for words_containing_underscores. And that feature's implementation (in redcarpet) uses [A-Za-z0-9]:

https://github.com/vmg/redcarpet/blob/master/ext/redcarpet/markdown.c#L613